test/helper/test_trigger.ts

```diff
@@ -641,7 +641,10 @@ const testTriggerFile = (file: string, info: TriggerSetInfo) => {
 
           for (const key of keys) {
             for (const param of outputStringsParams[key] ?? []) {
-              if (!Regexes.parse(`\\b${param}\\s*:`).exec(funcStr)) {
+              if (
+                (!Regexes.parse(`\\b${param}\\s*:`).exec(funcStr)) &&
+                (!Regexes.parse(`\\b${param.split('.')[0] ?? ''}\\s*:`).exec(funcStr))
+              ) {
```

Comment:
```suggestion
              const firstParam = param.split('.')[0] ?? param;
              if (!Regexes.parse(`\\b${param}\\s*:`).exec(funcStr)) {
```

I think this something along these lines is better than duplicating the regex.